crates/pop-cli/src/commands/up/rollup.rs

```diff
@@ -263,34 +565,144 @@ mod tests {
 			proxied_address: Some(MOCK_PROXIED_ADDRESS.to_string()),
 			..Default::default()
 		}
-		.prepare_for_registration(&mut cli)
+		.prepare_for_registration(&mut Deployment::default(), false, &mut cli)
 		.await?;
 
 		assert_eq!(chain_config.id, 2000);
-		assert_eq!(chain_config.genesis_code, genesis_code);
-		assert_eq!(chain_config.genesis_state, genesis_state);
+		assert_eq!(chain_config.genesis_artifacts.genesis_code_file, Some(genesis_code));
+		assert_eq!(chain_config.genesis_artifacts.genesis_state_file, Some(genesis_state));
 		assert_eq!(chain_config.chain.url, Url::parse(POLKADOT_NETWORK_URL)?);
 		assert_eq!(chain_config.proxy, Some(format!("Id({})", MOCK_PROXIED_ADDRESS.to_string())));
 		cli.verify()
 	}
 
+	#[tokio::test]
+	async fn prepare_for_deployment_works() -> Result<()> {
```

Comment:
Let's order the tests in the same order than fns are defined above

Reply:
Agree, bad habit. I've reordered it in https://github.com/r0gue-io/pop-cli/pull/459/commits/ed182a5ad9b8054dca76113d6deb6c6047e3db06